WordPress/src/main/res/values/reader_styles.xml

```diff
@@ -508,4 +508,12 @@
         <item name="android:ellipsize">end</item>
     </style>
 
+    <style name="ReaderTextView.Post.New.Removed.Title" parent="ReaderTextView">
+        <item name="android:ellipsize">marquee</item>
```

Comment:
Good point, I tested it now, and didn't work as well. Keep in mind I just copied the existing style for that Title and only changed the `font-family` and `textAppearance` to avoid introducing regressions, but since this doesn't work I can remove it as well.